lib/rules/require-return-from-computed.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const ember = require('../utils/ember');
+
+//------------------------------------------------------------------------------
+// General rule - Don't introduce side-effects in computed properties
+//------------------------------------------------------------------------------
+
+function isReachable(segment) {
+  return segment.reachable;
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Warns about missing return statements in computed properties',
+      category: 'Possible Errors',
+      recommended: false,
+      url: 'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-side-effects.md'
```

Comment:
This is the wrong URL. I'm not sure if this `url` property is actually used anywhere so you could just remove it.